### ISSUE TITLE
There are stocks that has an strange update

### ISSUE THREAD

**Issue by git-farid** [finnhubio/Finnhub-API#308]
## What is the issue ? Plz describe in details

HI , some hours ago(around 36) i sent an email to ask about when the data of each stock is included in the "historical data" section , as an alternative to the websocket because the transactions comes from IEX only and the data could be not complete , so i was looking for another way to has the "today's data" as soon as possible (i just have around 16 hrs to do what i need to do , so i need the today's data ASAP).

The answer that i got was:

**_"3. Tick data will be updated around 10-11PM EST time everyday."_**

But there are some stocks that are not updated in that hours range.

Take a look , see the dates and hours in my task bar(windows operative sistem) and the "total" index in the JSON file.

BA(Boeing)

1 image

![imagen](https://user-images.githubusercontent.com/62631387/86412714-312e1f80-bc85-11ea-9101-c6d90bf464e3.png)

2 image

![imagen](https://user-images.githubusercontent.com/62631387/86412897-8b2ee500-bc85-11ea-8fd1-f91478d99e2b.png)

image 3( at the next day)

![imagen](https://user-images.githubusercontent.com/62631387/86412820-6175be00-bc85-11ea-89f5-86a949c875b0.png)

image 4(updated data)

![imagen](https://user-images.githubusercontent.com/62631387/86412851-6dfa1680-bc85-11ea-8b7a-812a278f4507.png)

Looks like if it would be an aleatory order for this problem because the same thing happend for TSLA 24hrs before and , when i watch over around 14 stocks , beacuse i saw the same problem in TSLA , TSLA does not presents the same problem.

TSLA (When i sent the email)

![imagen](https://user-images.githubusercontent.com/62631387/86413569-22486c80-bc87-11ea-943b-0745141541dd.png)

TSLA today

![imagen](https://user-images.githubusercontent.com/62631387/86413630-4c019380-bc87-11ea-9951-5d57600f4ed3.png)

TSLA when i watch over the stocks to analize this problem (when i catched BA (Boeing)

image 1

![imagen](https://user-images.githubusercontent.com/62631387/86413765-be727380-bc87-11ea-82eb-57da05f13467.png)

image 2

![imagen](https://user-images.githubusercontent.com/62631387/86413989-55d7c680-bc88-11ea-9cd6-c219984d0eb9.png)

image 3 , TSLA at the next day , does not change the 'total' number in the JSON like BA did it. ,So in conclusion , TSLA presents the problem in 2020/06/30 , but at the next day , does not present the problem so this problem is aleatory??

![imagen](https://user-images.githubusercontent.com/62631387/86414100-a4856080-bc88-11ea-90b5-7d785b652853.png)

## How to reproduce the error ?

1-Just watch over the stocks (around 20) when before be updated (before 10PM , 11PM EST TIME)
2- Then watch the same stocks after be updated
3-Then watch the stocks at the next day (around 1 PM)

## What is the expected result and why do you think it's the correct result ?

What i expect is that all the stocks just change the results data 1 time , before 10PM , 11PM EST TIME , and after 10PM , 11PM EST TIME , no 2 times after 10PM , 11PM EST TIME.

**Comment by finnhubio** [comment]
It looks like you made the API call when we are updating the database. However let me have an engineer taking a look at this.

**Comment by git-farid** [comment]
Yes  , that is what i was doing , but the thing is that must be only 1 change in the data ,  the "not updated data" before the 10 PM EST Time , and the "updated data" after 10 PM -- 11 PM EST Time , but there are 3 changes in the data:

1-Before 10 PM EST Time
2-After 10 or 11 PM EST Time

3 -And then at the next day there is another update in the data , so that means that the first update is not the "real update"  , so it is not the "real data"(i guess) , is necesary wait until around the 1 PM ,  at the next day to see the "real update".

If the websocket does not provide all the transactions(because are only from IEX) , an if the "today's data" is not updated at the same day , well , basicly , there is not "today's data" service is only historical data.

_If there is something that i can do to help , tell me , i really need the "today's data" ASAP , everyday (at 10 PM or 11 PM EST Time ), it represents for me , "the last piece of my scanner" , without "today's data" i can not finish my software._

Thanks.

**Comment by git-farid** [comment]
(Just reporting)

Hi , i just take a look to BA in 2020/07/01 and appears like if the data would be another (again) , this time is not a significant number of transactions , but well , the number is different

![imagen](https://user-images.githubusercontent.com/62631387/86524439-6632a180-be40-11ea-8817-db98c466fb8f.png)

Compare it with the image number 4 where is the text  "image 4(updated data)"


![imagen](https://user-images.githubusercontent.com/62631387/86524485-c1649400-be40-11ea-93dd-d729e4ac4a43.png)


The same happens with others stocks , all without significant number of transaction
